diff -rupN /usr/lib/python2.7/dist-packages/provisioningserver/power_schema.py /usr/lib/python2.7/dist-packages/provisioningserver/power_schema.py
--- /usr/lib/python2.7/dist-packages/provisioningserver/power_schema.py	2015-10-14 19:39:11.000000000 +0300
+++ /usr/lib/python2.7/dist-packages/provisioningserver/power_schema.py	2016-01-12 14:01:19.724995051 +0200
@@ -313,4 +313,15 @@ JSON_POWER_TYPE_PARAMETERS = [
                 default='5', field_type='password'),
         ],
     },
+    {
+        'name': 'ravello',
+        'description': 'Ravello',
+        'fields': [
+            make_json_field('ravello_user', "Ravello user"),
+            make_json_field(
+                'ravello_password', "Ravello password", field_type='password'),
+            make_json_field('ravello_application', "Ravello application name"),
+            make_json_field('ravello_vm', "Ravello VM name"),
+        ],
+    },
 ]
diff -rupN /usr/lib/python2.7/dist-packages/provisioningserver/rpc/power.py /usr/lib/python2.7/dist-packages/provisioningserver/rpc/power.py
--- /usr/lib/python2.7/dist-packages/provisioningserver/rpc/power.py	2015-10-14 19:39:11.000000000 +0300
+++ /usr/lib/python2.7/dist-packages/provisioningserver/rpc/power.py	2016-01-12 13:59:44.472998383 +0200
@@ -75,6 +75,7 @@ QUERY_POWER_TYPES = [
     'ucsm',
     'virsh',
     'vmware',
+    'ravello',
 ]
 
 
